client/web/src/search/results/progress/StreamingProgress.scss

```
@@ -9,23 +9,14 @@
 
 .streaming-progress {
     &__count {
-        padding: 0.375rem 0.75rem; // Match padding for .btn (but this is not a button)
-        color: var(--link-color);
-
-        .theme-redesign & {
-            color: var(--body-color);
-            padding-left: 0.5rem;
-            padding-right: 0.5rem;
-            border-radius: var(--border-radius);
-
-            &--in-progress {
-                color: var(--link-color);
-            }
-        }
+        color: var(--body-color);
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
```

Missing top/bottom padding here.
```suggestion
        padding: 0.375rem 0.5rem
```